hawkular-agent-core/src/main/java/org/hawkular/agent/monitor/storage/AsyncInventoryStorage.java

```diff
@@ -201,6 +201,11 @@ private String getInventoryId(NamedObject no) {
             mb.unit(MetricUnit.valueOf(metricUnits.name()));
         }
         m.getProperties().forEach((k, v) -> mb.property(k, v.toString()));
+
+        // TODO remove these once h-metrics integration is removed
+        mb.property("hawkular.metric.type", m.getType().getMetricType().name());
+        mb.property("hawkular.metric.typeId", m.getType().getID().getIDString());
+        mb.property("hawkular.metric.id", m.getID().getIDString());
```

Comment:
added "hawkular.metric.id" - good @jotak ?

Reply:
This is good, think, but I also think @jotak is saying that "hawkular.metric.typeId" can go away.  We can wait for his approval or comment before merging.

Reply:
I just left type ID in there just in case it could be useful. We are removing all of these soon anyway, so I figured leave it in case type ID is useful and it'll go away eventually.

Reply:
LGTM